spark/src/main/java/io/ddf/spark/SparkDDFManager.java

```diff
@@ -272,8 +272,23 @@ public DDF copyFrom(DDF ddf) throws DDFException {
       if (ddf instanceof S3DDF) {
         S3DDF s3DDF = (S3DDF) ddf;
         S3DataSourceCredentials cred = s3DDF.getManager().getCredential();
-        uri = String.format("s3a://%s:%s@%s/%s", cred.getAwsKeyID(), cred.getAwsScretKey(),
-            s3DDF.getBucket(), s3DDF.getKey());
+        if (s3DDF.getPath() != null) {
+          String[] fileList;
+          String path = s3DDF.getPath();
+          fileList = path.split(",");
```

Comment:
adatao-sample-data/test/parquet/wildcard/{part=1,part=2}/ will help users, but it also brings some trouble. for example, should we support adatao-sample-data/test/parquet/{date1, date2}/{part=1, part=2}?If it's in arbitary format, it's really difficult for us to handle correctly. I suggest we only allow the easiest "full path, full path"